integrations/huggingface-transformers/README.md

```diff
@@ -14,46 +14,80 @@ See the License for the specific language governing permissions and
 
-## SparseML CLI
+## Tutorials
+
+- [Sparse Transfer Learning](sparse-transfer-learning-bert.md) [**RECOMMENDED**]
+- Sparsification from Scratch (example coming soon!)
+
+### Use Case Examples - CLI
+- [Sparse Transfer Learning for Sentiment Analysis](tutorials/sentiment-analysis/sentiment-analysis-cli.md)
+- [Sparse Transfer Learning for Text Classification](tutorials/text-classification/text-classification-cli.md)
+- [Sparse Transfer Learning for Token Classification](tutorials/token-classification/token-classification-cli.md)
+- [Sparse Transfer Learning for Question Answering](tutorials/question-answering/question-answering-cli.md)
+- Sparsifying from Scratch (example coming soon!)
+
+### Use Case Examples - Python
+
+#### Sentiment Analysis (Single Input Binary Text Classification)
+- [Sparse Transfer with GLUE Datasets (SST2) for sentiment analysis](tutorials/sentiment-analysis/docs-sentiment-analysis-python-sst2.ipynb)
+- [Sparse Transfer with Custom Datasets (RottenTomatoes) and Custom Teacher from HF Hub for sentiment analysis](tutorials/sentiment-analysis/docs-sentiment-analysis-python-custom-teacher-rottentomatoes)
+
+#### Text Classification (Single / Multi Input Text Classification)
+- [Sparse Transfer with GLUE Datasets (QQP) for multi-input text classification](tutorials/text-classification/docs-text-classification-python-qqp.ipynb)
+- [Sparse Transfer with Custom Datasets (SICK) for multi-input text classification](tutorials/text-classification/docs-text-classification-python-sick.ipynb)
+- [Sparse Transfer with Custom Datasets (TweetEval) and Custom Teacher for single input text classificaiton](tutorials/text-classification/docs-text-classification-python-custom-teacher-tweeteval.ipynb)
+- [Sparse Transfer with Custom Datasets (GoEmotions) for multi-label text classification](tutorials/text-classification/docs-text-classification-python-multi-label-go_emotions.ipynb)
+
+#### Token Classification 
+- [Sparse Transfer with Conll2003 for named entity recognition](tutorials/token-classification/docs-token-classification-conll2003.ipynb)
+- [Sparse Transfer with Custom Datasets (WNUT) and Custom Teacher for named entity recognition](tutorials/token-classification/docs-token-classification-custom-teacher-wnut.ipynb)
+
+#### Question Answering
 
-The SparseML installation provides a CLI for sparsifying your models for a specific task; appending the `--help` argument will provide a full list of options for training in SparseML:
+- Sparse Transfer with SQuAD (Example coming soon!)
+- Sparse Transfer with Squadshifts Amazon (Example coming soon!)
+
+## Quick Tour
+
+### SparseZoo
+
+Neural Magic has pre-sparsified many common models, including BERT-base, BERT-large, DistillBERT, and RoBERTa. These models and associated sparsification recipes can be deployed directly or can be fine-tuned onto custom dataset via sparse transfer learning. This makes it easy to create a sparse version of model trained on your dataset.
```

Comment:
version of model trained on your dataset -> version of the model trained on your dataset